devices/ePaper/Drivers/JD796xx/JD79653A.cs

```diff
@@ -0,0 +1,664 @@
+﻿// Copyright (c) 2022 The nanoFramework project contributors
+// See LICENSE file in the project root for full license information.
+
+using System;
+using System.Device.Gpio;
+using System.Device.Spi;
+using System.Drawing;
+using System.Threading;
+using Iot.Device.EPaper.Buffers;
+using Iot.Device.EPaper.Enums;
+using nanoFramework.UI;
+
+namespace Iot.Device.EPaper.Drivers.JD796xx
+{
+    /// <summary>
+    /// Base class for GDEW0154-based ePaper devices.
+    /// </summary>
+    public abstract class JD79653A : IEPaperDisplay
```

Comment:
Technically the name should be `Jd79653A` to follow our guidelines. Same for the namespace `Jd796xx`